tap_mssql/sync_strategies/common.py

```diff
@@ -87,16 +95,26 @@ def get_key_properties(catalog_entry):
     return key_properties
 
 
-def generate_select_sql(catalog_entry, columns):
+def generate_select_sql(catalog_entry, columns, fastsync=False):
     database_name = get_database_name(catalog_entry)
     escaped_db = escape(database_name)
     escaped_table = escape(catalog_entry.table)
     escaped_columns = [escape(c) for c in columns]
 
-    select_sql = "SELECT {} FROM {}.{}".format(
-        ",".join(escaped_columns), escaped_db, escaped_table
-    )
+    if fastsync:
```

Comment:
oh yeah good call.  I planned on having a condition within full_table.sync_table() that would call one or the other, so that was something i meant to do but forgot to make note of.  This is updated

Reply:
oops, i think i read 'folder' as 'function' lol